README.md

```diff
@@ -1,5 +1,51 @@
 # ensembl-client
 
-This repository provides an all-new frontend for the Ensembl website, written in React, Rust and WASM. The current public release can be seen at https://2020.ensembl.org
+## Introduction
 
-If you are interested in the latest developments in this project, please join our Slack channel #ensembl2020
+Ensembl is an open-source, open-access bioinformatics project based at [EMBL-EBI](https://www.ebi.ac.uk) which creates, integrates and distributes genomic data and tools. The main entry point is our website, which provides visualisations of our genomic data as well as online tools for analysing your own data.
+
+This repository is an all-new frontend for the Ensembl website, written in React, Rust and WASM. The current public release can be seen at https://2020.ensembl.org. The new site will integrate genomes from across the taxonomic space, including vertebrates, invertebrates, plants, fungi and bacteria.
+
+## Architecture
+
+The Ensembl client consists of a number of separate apps, each of which relies on one or more backend services to supply content. It is our intention that the Ensembl client code will be reusable in part or whole by other websites.
+
+The following apps are currently in development:
+
+* Genome Browser
+* Species Selector
+* Entity Viewer
+* Custom Download
+* Global Search
+
+Additional apps will become available as the site expands.
+
+In addition to the main web frontend, the codebase includes an installation of [Storybook](https://storybook.js.org/) which can be used to examine individual components.
```

Comment:
Perhaps the link to our storybook will be more informative here than the link to the site of the storybook project?

Reply:
There are now links to our storybook as well.